src/adjustItems.js

```diff
@@ -0,0 +1,25 @@
+function addTodo(arr) {
+  const textGetter = document.getElementById('TodoDesc');
+  arr.push({
+    description: textGetter.value,
+    completed: false,
+    index: arr.length,
+  });
+  textGetter.value = '';
+}
+function deleteTodo(arr, index) {
+  const cont = document.getElementById(index);
+  cont.remove();
+  arr.splice(index, 1);
+  localStorage.clear();
+  localStorage.setItem('todoArray', JSON.stringify(arr));
+}
```

Comment:
* I suggest you add a function for editing tasks description as stated in the [Project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20editing%20task%20descriptions.) currently the feature hasn't been implemented yet 🙏

Reply:
Hello, the feature is implemented but is not a function. It's using the contentEditable tag to work instead. You can try it by clicking on the description text.

Reply:
* Oh 😮 sorry i missed that 🙏 you can go ahead and implement the other changes 👍

Reply:
They're all done now as well, thank you for your time.

Reply:
Unfortunately, this feature doesn't work as expected. It is impossible to change the description of the first item (It doesn't update local storage and keeps the old description when the page is refreshed). And also as written in the project specification having a function for is a requirement.

https://user-images.githubusercontent.com/1642014/126027914-b2cc739a-030a-4e1e-8fcf-93da6e7de29a.mp4

Reply:
The edit function doesn't accept ENTER as submit, but a click that is outside of the text area. I can change it if it's a requirement but if it's OK as is, I wouldn't want to risk breaking something new as this was my second review request.

Reply:
Edit function added as well, still working in the same way as before.

Reply:
Edit function now works on both "Enter" click and click out of focus.